src/lib/apiClient.js

```diff
@@ -99,9 +99,6 @@ const createCloudCodeFunction = (bastionName, func, callback) => {
     method: 'post',
     data: func,
     url: `${baseUrl}/ccf/${bastionName}`,
-    headers: {
-      'Content-Type': 'application/json',
-    }
```

Comment:
I believe this was breaking uploading ccfs.  Seems to be working now.  Delete is not yet working